src/www/ui/api/documentation/openapi.yaml

```
@@ -1735,6 +1822,34 @@ components:
             - INFO
             - ERROR
           description: Denotes if info was created on error
+    Token:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Token Id
+          example: 2.3
+        name:
+          type: string
+          description: Name of the Token
+        created:
+          type: string
+          description: Denotes when the token was created
+          example: 2022-07-11
+        expire:
+          type: string
+          description: Denotes when the token was created
+          example: 2022-07-11
+        scope:
+          type: string
+          enum:
+          - r
+          - w
```

Please change name to be consistent with token creation request.
```suggestion
        scope:
          type: string
          enum:
          - read
          - write
```

---

okay!